wp-moj-elasticsearch.php

```diff
@@ -3,44 +3,35 @@
  * Plugin name: WP MoJ ElasticSearch
  * Plugin URI:  https://github.com/ministryofjustice/wp-moj-elasticsearch
  * Description: MoJ WP plugin to extend the functionality of the ElasticPress plugin
- * Version:     1.3.1
+ * Version:     2.0.0
  * Authors:     Damien Wilson, Adam Brown
  * Text domain: wp-moj-elasticsearch
  * Author URI:  https://ministryofjustice.github.io/justice-on-the-web
  * License:     MIT License
  *
  **/
 
-defined('ABSPATH') or die('No humans allowed.');
-// Debug output related constant
-define('DEBUG_ECHO', false);
-
-// Check WP hasn't malfunctioned is ready to go.
-if (!function_exists('add_action')) exit;
-
-// Get site root to target installation of Composer's autoloader
-global $root_dir;
-
-if (empty($root_dir)) {
-    trigger_error(
-        'WP MoJ ElasticSearch expects Bedrock. 
-        For your project add $root_dir = dirname(__DIR__); in your wp-config.php file.',
-        E_USER_WARNING
-    );
+if (!defined('MOJ_ROOT_DIR')) {
+    header("HTTP/1.1 403 Forbidden");
     return;
 }
 
```

Comment:
I would leave a short one line comment to say where MOJ_ROOT_DIR is defined? Like "MOJ_ROOT_DIR is defined in the theme config/application.php" or something like that. I know where it is, but I'm not sure other devs coming to the project would know.